bin/thoth

```diff
@@ -3889,6 +3889,12 @@ var main = function ()
 		process.env.DEBUG = '1';
 	}
 
+	/* see https://github.com/joyent/node-exeunt */
```

Comment:
In what way is it not clear?

Reply:
The commit message says "forward porting this fix", but I'm not sure what was wrong, how this fixes it, etc. I don't know what I'm looking for in the node-exeunt repo that pertains to this code block.

I'm not considering this one a blocker, but just pointing out that I have more questions from the comment than I do answers. Additional information would be helpful (to me at least), but I don't consider it necessary.

Reply:
Comment from @jlevon below:

> For some reason I can't reply to your exact comment, but I'm not planning to remove the --abort flag right now. AIUI it doesn't break Mac, and I see no reason to try to support Linux explicitly.

That's sufficient for me, so I'll mark this as resolved.

Reply:
Would it help if I said "see the README" ? Guess it wasn't obvious you were supposed to just read that from clicking the link?

Reply:
I guess what's tripping me up is that since node-exeunt isn't a dependency, why it's actually related.

Reply:
Could perhaps point directly to https://github.com/joyent/node-exeunt#solution-3-set-stdoutstderr-to-be-blocking because, IIUC, this is choosing to use that solution.

Reply:
Yeah, that does help a bit. So we're not including this as a dependency, but implementing the same thing as solution 3.

Reply:
I'll deep link like that then, thanks.

Reply:
Yes, works for me. Thanks!!